invenio_records_resources/records/systemfields/files/manager.py

```diff
@@ -156,6 +157,12 @@ def create(self, key, obj=None, stream=None, data=None):
         if stream:
             obj = ObjectVersion.create(self.bucket, key, stream=stream)
         if obj:
+            if isinstance(obj, dict):
```

Comment:
Not the cleanest of the ways. Accessing DB models at this level is necessary due to the strictness of both `ObjectVersion` and `FileInstance` _create_ methods. 

It should be evaluated if this is "acceptable" enough. Afterall, there is no `File API` class (i.e. and `api.py` in files-rest). Therefore, this system field acts as one.